docs/source/release_notes/lts/52_0.rst

```diff
@@ -0,0 +1,162 @@
+  reliable method (libvirt/qemu-gust-agent query).
+- Set ``LC_ALL=C`` by default on sysinfo collection to simplify avocado
+  diff comparison between different machines.
+- Result plugins system is now pluggable and the results plugins (JSON,
+  XUnit, HTML) were turned into stevedore plugins. They are now listed
+  in the ``avocado plugins`` command.
+- Multiplexer was replaced with Varianter plugging system with defined
+  API to register plugins that generate test variants.
+- Old ``--multiplex`` argument, which used to turn yaml files into
+  variants, is now handled by an optional plugin called ``yaml_to_mux``
+  and the ``--multiplex`` option is being deprecated in favour of
+  the ``--mux-yaml`` option, which behaves the same way.
+- It’s now possible to disable plugins by using the configuration file.
+- Better error handling of the virtual machine plugin (``--vm-domain``
+  and related options).
+- When discovering tests on a directory, the result now is a properly
+  alphabetically ordered list of tests.
+- Plugins can now be setup in Avocado configuration file to run at a
+  specific order.
+- Support for filtering tests by user supplied "tags".
+- Users can now see the tags set on `INSTRUMENTED` test when listing
+  tests with the `-V` (verbose) option.
+- Users can now choose to keep the complete set of files, including
+  temporary ones, created during an Avocado job run by using the
+  ``--keep-tmp`` option (e.g. to keep those files for ``rr``).
+- Tests running with the external runner (``--external-runner``) feature
+  will now have access to the extended behavior for SIMPLE tests, such
+  as being able to exit a test with the WARNING status.
+- Encoding support was improved and now Avocado should safely treat
+  localized test-names.
+- Test writers now have access to a test temporary directory that will
+  last not only for the duration of the test, but for the duration of
+  the whole job execution to allow sharing state/exchanging data between
+  tests.
+- When not disabled, "results.tap" is by default generated in results
+  directory.
+- Avocado is now available on Fedora standard repository. The package
+  name is ``python2-avocado``. The optional plugins and examples
+  packages are also available. Run ``dnf search avocado`` to list them
+  all.
+- Optional plugins and examples packages are also Avocado on PyPI.
+- Avocado test writers can now use a family of decorators, namely
+  ``avocado.skip()``, ``avocado.skipIf()`` and ``avocado.skipUnless()``
+  to skip the execution of tests.
+- ``self.skip()`` is under deprecation process and users should use the
+  skip decorators instead.
+- Sysinfo collection based on command execution now allows a timeout to
+  be set.
+- The non-local runner plugins, the html plugin and the yaml_to_mux
+  plugin are now distributed in separate packages.
+- Users sending SIGTERM to Avocado can now expect it to be properly handled.
```

Comment:
What does it mean to be properly handled? (hint: Killed Avocado process now also tries to kill all the test processes)